### ISSUE TITLE
Collections only show one instance of a recurring event

### ISSUE THREAD

**Issue by cdw9** [plone/Products.CMFPlone#3648]
### What I did:

* create an event with recurrence
* create a collection, with the only criteria being to display items with type = event

### What I expect to happen:

I expect the collection would display all occurrences of the event separately

### What actually happened:

Only the one event is displayed

### What version of Plone/ Addons I am using:

Tested on the Plone 5 and Plone 6 demos

**Comment by davisagli** [comment]
There is a special `event_listing` view which includes the recurrences: https://github.com/plone/plone.app.event/blob/master/plone/app/event/browser/configure.zcml#L25

(Normally collections simply query the catalog and each result is a different content item. Event recurrences need to be queried in a different way.)

**Comment by cdw9** [comment]
> There is a special event_listing

Yes, but if you want to build a custom listing based on some other condition, you can't do that TTW.

I realize this may be more of a feature request, but wanted to throw it out there.

**Comment by davisagli** [comment]
@cdw9 It looks like the event_listing view implementation (https://github.com/plone/plone.app.event/blob/master/plone/app/event/browser/event_listing.py) has some code to use the query from a collection as the starting point for looking up the event recurrences. But, I haven't tried it myself.

**Comment by jensens** [comment]
Closing as outdated (4 years old). Collections showing only one recurring event instance is by-design per davisagli. The event_listing view handles recurrences separately.
If you think this issue is still relevant please reopen it. Thank you for the original report!
